code/__DEFINES/bots.dm

```diff
@@ -24,6 +24,8 @@
 #define BOT_NAV				15	// computing navigation
 #define BOT_WAIT_FOR_NAV	16	// waiting for nav computation
 #define BOT_NO_ROUTE		17	// no destination beacon found (or no route)
+#define BOT_MAKETILE		18	// converting metal into tiles (floorbots)
+#define BOT_EATTILE			19	// adding said tiles to inventory (floorbots)
```

Comment:
```suggestion
#define BOT_MAKE_TILE		18	// converting metal into tiles (floorbots)
#define BOT_EAT_TILE			19	// adding said tiles to inventory (floorbots)
```
Can you snake case these defines like this? You should use an underscore between each word in proper snake case.